website/blog/2023-01-17-ferretdb-0-8-1-version-release.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "FerretDB new version release - 0.8.1"
+slug: ferretdb-0-8-1-version-release
+author: Alexander Fashakin
+description: "FerretDB 0.8.1 - the open-source MongoDB alternative - includes new features like version availability, `distinct` command & client TLS validation, and much more."
+image: ../static/img/blog/FerretDB_v0.8.1.jpg
+tags: [release]
+---
+
+![FerretDB version 0.8.1](../static/img/blog/FerretDB_v0.8.1.jpg)
+
+<!--truncate-->
+
+[FerretDB](https://www.ferretdb.io/) 0.8.1 is here, and it's better than ever!
+On the back of our Beta release, we've gone even further to improve FerretDB with lots of new features and enhancements.
+This FerretDB release and the continued support of our amazing community further strengthens our drive to bring you the open source alternative to MongoDB.
+And now, as we approach our GA version later this quarter, we’re more emboldened to ensure FerretDB matches up with the behaviour you’d expect from MongoDB, and to create more compatibility with other applications and real world use cases.
+
+Keep reading to learn more.
+
+## What's new
+
+We'll start off with probably the most anticipated update!
+We are delighted to announce that we now report the availability of newer versions of FerretDB in mongosh.
+However, this is only available for users with telemetry enabled.
+If telemetry is enabled and a newer version of FerretDB is available, you'll be notified and can stay up-to-date with the latest and best version of FerretDB.
+
+If you are yet to enable telemetry and you’d love to access this feature, please [see our documentation here](https://docs.ferretdb.io/telemetry/).
+
+Next, we've implemented the `distinct` command in FerretDB.
+With this command, you can easily find the unique values of a specific key in your data.
+The `distinct` command takes three arguments: `distinct`, `key`, and `query`.
+For instance, if you want to find the unique values of the "age" field in a specific "people" collection, run the command below:
+
+```js
+db.runCommand( { distinct: "people", key: "age", query: {} } )
```

Comment:
The command you offer is possible, but maybe it might make sense to offer `distinct` itself (not `runCommand`):
```
db.people.distinct("age", {})
```